AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerServiceImpl.kt

```diff
@@ -204,6 +181,61 @@ internal class TestRunnerServiceImpl internal constructor(
         }
     }
 
+    private suspend fun findLogcats(
+        testMatrix: TestMatrix
+    ): MutableMap<TestRunnerService.TestIdentifier, TestRunnerService.ResultFileResource> {
+        var testCaseLogcatBlobs = mutableMapOf<TestRunnerService.TestIdentifier, TestRunnerService.ResultFileResource>()
+        val resultPath = testMatrix.resultStorage.googleCloudStorage.gcsPath
```

Comment:
it is best to convert this to GcsPath value class so we can have 1 central way to deal with them.
you end up converting it back anyways so it is better to do it and use class operations instead.